swabbie-core/src/main/kotlin/com/netflix/spinnaker/config/SwabbieProperties.kt

```diff
@@ -25,12 +25,6 @@ import java.time.LocalDateTime
 import java.time.LocalTime
 import java.time.ZoneId
 
-/**
```

Comment:
Why did you delete this comment? I think it's helpful.

Reply:
I was feeling like deleting some comments I guess. Will put them back.

Reply:
Thank you!